src/components/pages/home/HomeLayout.jsx

```diff
@@ -0,0 +1,41 @@
+import MainContent from './MainContent';
+import CategoryButtons from './category-btns/CategoryButtons';
+import DeviceDetailContent from './DeviceDetailsContent';
+import { Switch, Route, useRouteMatch } from "react-router-dom";
+
+
+import { makeStyles } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+
+const useStyles = makeStyles((theme) => ({
+   homeContainer: {
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      margin: '3vh 5vh 3vh 5vh',
+   }
+}));
+
+const HomeLayout = () => {
+   const classes = useStyles();
+   let { path } = useRouteMatch();
+   const truePath = (path === '/') ? '/home' : path;
+   return (
+      <Box className={classes.homeContainer}>
+         <CategoryButtons />
+         <Switch>
```

Comment:
possibly nest routing in app.jsx instead of here

Reply:
Yes all routing needs to be done in the home page. Once it gest too big, it'll be hard to find where all the routings are if it's not centralized.